apps/code-examples/src/app/code-examples/forms/input-box/basic/input-box-demo.component.html

```diff
@@ -1,29 +1,137 @@
-<div [skyTheme]="modernLightTheme">
-  <h3>With basic HTML elements</h3>
-  <sky-input-box>
-    <label class="sky-control-label" for="myInput"> HTML &lt;input&gt; </label>
-    <input class="sky-form-control" id="myInput" type="text" />
-  </sky-input-box>
-  <sky-input-box>
-    <label class="sky-control-label" for="myTextarea">
-      HTML &lt;textarea&gt;
-    </label>
-    <textarea class="sky-form-control" id="myTextarea"></textarea>
-  </sky-input-box>
-
-  <h3>With SKY UX components</h3>
-  <sky-input-box>
-    <label class="sky-control-label" for="myDatepicker"> Datepicker </label>
-    <sky-datepicker>
-      <input id="myDatepicker" skyDatepickerInput />
-    </sky-datepicker>
-  </sky-input-box>
-  <sky-input-box>
-    <label class="sky-control-label" for="myInlineHelp"> Inline Help </label>
-    <sky-help-inline
-      class="sky-control-help"
-      (actionClick)="onActionClick()"
-    ></sky-help-inline>
-    <input class="sky-form-control" id="myInlineHelp" />
-  </sky-input-box>
+<div
+  [ngStyle]="{
+    maxWidth: '800px'
+  }"
+>
+  <form [formGroup]="newMemberForm">
+    <sky-fluid-grid [disableMargin]="false" gutterSize="small">
+      <sky-row>
+        <sky-column [screenSmall]="12">
+          <h2>New Member Form</h2>
```

Comment:
```suggestion
          <h2>New member form</h2>
```
It's minor, but our examples should follow our guidance to use sentence-style capitalization in the UI.